orbit/cmd/desktop/desktop.go

```diff
@@ -23,6 +26,35 @@ import (
 // version is set at compile time via -ldflags
 var version = "unknown"
 
+func setupRunners() {
+	var runnerGroup run.Group
+
+	// Install a signal handler runner
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	var execFn func() error
+	var interrFn func(error)
+
+	if runtime.GOOS == "windows" {
+		execFn, interrFn = run.SignalHandler(ctx, os.Interrupt, os.Kill)
+	} else {
+		execFn, interrFn = run.SignalHandler(ctx, os.Interrupt, os.Kill, syscall.SIGTERM)
+	}
+
+	runnerGroup.Add(
+		execFn,
+		func(err error) {
+			interrFn(err)
+			systray.Quit()
+		},
+	)
+
+	if err := runnerGroup.Run(); err != nil {
+		return
```

Comment:
Let's log `err` here.

Reply:
thanks! I copy pasted this code from orbit. I initially had a log here but then removed it for some reason. I've just re-added the err log

Reply:
> I've just re-added the err log

Maybe you missed to push the commit?

Reply:
that's correct :) Just pushed the change